### PR TITLE
Added Flagsmith datadog widget

### PR DESCRIPTION
**Pull request by dabeeeenster**
### What does this PR do?

Adds the Flagsmith dashboard widget to our Flagsmith integration tile.

### Motivation

We've had the Flagsmith widget approved and want to communicate it.

### Review checklist

- [X] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [X] Feature or bugfix has tests
- [X] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Nothing!

### CODE REVIEW

File: flagsmith/README.md [hunk 1]
```diff
@@ -34,13 +38,26 @@ Feature flag tracking is available in the RUM Browser SDK. For detailed set up i
...
+        trackTraits: true,
+    },
+    ...
+})
 ```
 
+### Flagsmith Dashboard widget
+
+1. On the [Flagsmith integration tile](https://app.datadoghq.com/integrations/flagsmith), make sure the integration is installed.
+1. Make sure you are logged into Flagsmith with the account you want to see in Datadog.
```
Comment (rtrieu) [inline flagsmith/README.md:51]:
```suggestion
2. Make sure you are logged into Flagsmith with the account you want to see in Datadog.
```

File: flagsmith/README.md [hunk 2]
```diff
@@ -34,13 +38,26 @@ Feature flag tracking is available in the RUM Browser SDK. For detailed set up i
...
+1. In Datadog, navigate to an existing dashboard or create a new one.
```
Comment (rtrieu) [inline flagsmith/README.md:52]:
```suggestion
3. In Datadog, navigate to an existing dashboard or create a new one.
```

File: flagsmith/README.md [hunk 3]
```diff
@@ -34,13 +38,26 @@ Feature flag tracking is available in the RUM Browser SDK. For detailed set up i
...
+1. Press the **Add Widgets** button to expose the widget drawer.
```
Comment (rtrieu) [inline flagsmith/README.md:53]:
```suggestion
4. Press the **Add Widgets** button to expose the widget drawer.
```

File: flagsmith/README.md [hunk 4]
```diff
@@ -34,13 +38,26 @@ Feature flag tracking is available in the RUM Browser SDK. For detailed set up i
...
+1. Search for **Flagsmith** to find the Flagsmith widget in the **Apps** section of the widget drawer.
```
Comment (rtrieu) [inline flagsmith/README.md:54]:
```suggestion
5. Search for **Flagsmith** to find the Flagsmith widget in the **Apps** section of the widget drawer.
```

File: flagsmith/README.md [hunk 5]
```diff
@@ -34,13 +38,26 @@ Feature flag tracking is available in the RUM Browser SDK. For detailed set up i
...
+1. Select the **Flagsmith widget icon** to add it your your dashboard and open the **Flagsmith editor** modal. You can choose to add either the Flag or Audit log viewer widget.
```
Comment (rtrieu) [inline flagsmith/README.md:55]:
```suggestion
6. Select the **Flagsmith widget icon** to add it your your dashboard and open the **Flagsmith editor** modal. You can choose to add either the Flag or Audit log viewer widget.
```

File: flagsmith/README.md [hunk 6]
```diff
@@ -34,13 +38,26 @@ Feature flag tracking is available in the RUM Browser SDK. For detailed set up i
...
+1. Select the Flagsmith Organisation, Project and Environment you want to add to your dashboard.
```
Comment (rtrieu) [inline flagsmith/README.md:56]:
```suggestion
7. Select the Flagsmith Organization, Project and Environment you want to add to your dashboard.
```

File: flagsmith/README.md [hunk 7]
```diff
@@ -34,13 +38,26 @@ Feature flag tracking is available in the RUM Browser SDK. For detailed set up i
...
+1. Once selected, copy and paste the **Project ID** and **Environment ID** into Datadog.
```
Comment (rtrieu) [inline flagsmith/README.md:57]:
```suggestion
8. Once selected, copy and paste the **Project ID** and **Environment ID** into Datadog.
```

File: flagsmith/README.md [hunk 8]
```diff
@@ -34,13 +38,26 @@ Feature flag tracking is available in the RUM Browser SDK. For detailed set up i
...
+1. Select the page size and, optionally, a widget title and Flagsmith Tag to filter on.
```
Comment (rtrieu) [inline flagsmith/README.md:58]:
```suggestion
9. Select the page size and, optionally, a widget title and Flagsmith Tag to filter on.
```

File: flagsmith/README.md [hunk 9]
```diff
@@ -34,13 +38,26 @@ Feature flag tracking is available in the RUM Browser SDK. For detailed set up i
...
+1. Click **Save** to finish configuring the dashboard widget.
```
Comment (rtrieu) [inline flagsmith/README.md:59]:
```suggestion
10. Click **Save** to finish configuring the dashboard widget.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Some minor suggestions, but otherwise looks good.

**Comment by dabeeeenster** [comment]
OK thanks all these suggestions are committed

**Review by rtrieu — APPROVED** [review]
Hey @dabeeeenster sorry for not catching this before, but FYI I fixed and reordered the links, which should follow our [relative link convention](https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#Links), otherwise they may break in production.

**Review by emarsha94 — APPROVED** [review]

**Review by emarsha94 — APPROVED** [review]

**Pull request merged**